### PR TITLE
#19 - added docker image

### PR DESCRIPTION
**Pull request by wochinge**
**Motivation**:
Build a lean standalone Docker image for the rasa core sdk which can be 
* enhanced easily
* connected to a rasa core contaner
* required to implement https://github.com/RasaHQ/rasa_core/issues/845

**Proposed changes**:
- adding Dockerfile including an entrypoint file for simpler execution.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+FROM python:2.7-slim
```
Comment (tmbo) [inline Dockerfile]: should be 3.6 - I think we shouldn't use 2 in any of our docker files

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  apt-get install -y --no-install-recommends \
+  build-essential && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
+  mkdir /app
+
+WORKDIR /app
+
+# Copy as early as possible so we can cache ...
+ADD requirements.txt .
```
Comment (tmbo) [inline Dockerfile]: i think `COPY` is the better choice here (as we don't need fancy `ADD` features)

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+RUN pip install -r requirements.txt --no-cache-dir
+
+ADD . .
```
Comment (tmbo) [inline Dockerfile]: `COPY`

File: entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
```
Comment (tmbo) [inline entrypoint.sh]: let's be even more strict and use `set -Eeuo pipefail`

File: entrypoint.sh [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    start)
+        exec python -m rasa_core_sdk.endpoint "${@:2}"
+        ;;
+    run)
+        exec "${@:2}"
+        ;;
+    *)
+        print_help
+        ;;
+esac
```
Comment (tmbo) [inline entrypoint.sh]: newline

File: README.md [hunk 6]
```diff
@@ -24,7 +24,41 @@ pip install rasa_core_sdk
...
+Before starting the action server ensure that the folder containing your actions is handled as Python module and therefore has to contain a file called `__init__.py`
+
+Then start the action server using:
+
+```bash
+docker run --mount type=bind,source=<ABSOLUTE_PATH_TO_YOUR_ACTIONS>,target=/app/actions \
+	rasa/rasa_core_sdk:latest 
+```
+
+The action server is then avaible at `http://localhost:5055/webhook`.
```
Comment (tmbo) [inline README.md]: aren't we missing a port mapping from inside the container to the host machine? otherwise this webhook can't be reached

**Comment by tmbo** [comment]
once merged, this should be build on docker hub

File: README.md (hunk 6 above)
Comment (wochinge) [reply README.md]: added port mapping

**Pull request merged**